From 6e9cea3f783129e999ed7527706c20b565d0cc60 Mon Sep 17 00:00:00 2001
From: Zhang Ying-22455 <ying.zhang22455@nxp.com>
Date: Fri, 25 May 2018 11:14:40 +0800
Subject: [PATCH 465/741] arm64: dts: ls1012a: Add the identify of the platform
 to support to set rcpm bit

commit aa9c60f89646f3dbad65a8317b21cb9ae2c4e26f from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

Add the identify of the platform to support set the rcpm with
big-endian or little-endian.

Signed-off-by: Zhang Ying-22455 <ying.zhang22455@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi
index a7d8f90767d1..1eaa382a6432 100644
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a.dtsi
@@ -289,10 +289,10 @@
 		};
 
 		ftm0: ftm0@29d0000 {
-			compatible = "fsl,ftm-alarm";
+			compatible = "fsl,ls1012a-ftm-alarm";
 			reg = <0x0 0x29d0000 0x0 0x10000>,
 			      <0x0 0x1ee2140 0x0 0x4>;
-			reg-names = "ftm", "FlexTimer1";
+			reg-names = "ftm", "pmctrl";
 			interrupts = <0 86 0x4>;
 			big-endian;
 		};
-- 
2.17.1

